src/getter.c

```diff
@@ -96,10 +106,25 @@ int get_once(struct worker *workers, char **urls_opt, size_t urls_l, char *urls_
 				if(sscanf(buf, "OK %d bytes", &bytes) == 1) {
 					total_bytes += bytes;
 					reqs++;
+				        w->status = STATUS_READY;
 				} else if(sscanf(buf, "ERR %d", &err) == 1) {
 					fprintf(stderr, "cURL error: %s for URL '%s'.\n", curl_easy_strerror(err), w->url);
-				}
-				w->status = STATUS_READY;
+				        w->status = STATUS_READY;
+				} else if(sscanf(buf, "REP %d", &dl_delta) == 1) {
+                                         report_bytes_dl += dl_delta;
+                                         gettimeofday(&now_tv, NULL);
+                                         now = now_tv.tv_sec + now_tv.tv_usec / 1000000.0;
+                                         time_delta = now - last_report;
+                                         if(time_delta >= opt->worker_report_interval) {
+                                           speed = report_bytes_dl / time_delta;
+                                           fprintf(stderr, "GETTER_INTERIM_RESULT[%u]=%f\r\n", report_count, speed); 
+                                           fprintf(stderr, "GETTER_INTERVAL[%u]=%f\r\n", report_count, time_delta);
+                                           fprintf(stderr, "GETTER_ENDING[%u]=%f\r\n", report_count, now);
```

Comment:
This is going to be garbled if there is more than one active worker. I.e., you are going to get values from alternating values from different workers, which means that the interval you calculate here will be wrong, and the values will be misleading. You'll need to have a separate series for each worker, so do the interval calculation in the worker thread and pass that along with the value to the reporter thread, then add a worker ID to the output.

Also, I'm not particularly fond of netperf's output format for this. I'd prefer if you output each data point on a single line in a format that works for both human and machine consumption. Something like:

```Worker 1: 400 bytes over 0.254 seconds ending at 1507885098.234```

Or you can output the speed instead of bytes, of course. Don't have any strong opinions on that... And feel free to tweak the exact output format as well if you want; the most important is keeping each data point on a single line.